allennlp/commands/__init__.py

```diff
@@ -6,13 +6,30 @@
 from allennlp.commands.train import Train
 from allennlp.commands.evaluate import Evaluate
 from allennlp.commands.subcommand import Subcommand
+from allennlp.service.predictors import DemoModel
 
-# a mapping from predictor `type` to the location of the trained model of that type
+# This maps from the name of model (as known to the front-end)
```

Comment:
"This maps from the name of the task"?

Reply:
Right now it happens to be the name of the task, but in the future it might not be. 

For example, imagine at some point we want to put two different SRL models in our demo. then both will be "semantic-role-labeling" as the task, but they'll need to hit different

```
/predict/name-of-model
```

endpoints, like 

```
/predict/semantic-role-labeling-sota
/predict/semantic-role-labeling-simple
```

I don't know exactly what to call that. "model" doesn't feel 100% right but neither does "task"

Reply:
`/predict/semantic-role-labeling/sota`?

Reply:
yeah, that makes sense, but it will require some more code changes if/when we do it

Reply:
Not that you should make this change now, but I think eventually it makes sense to have this split between task and model, so it'll be easier to have comparisons between models, and put your own in-development model into a live demo.